src/test/Components/Quote.test.js

```diff
@@ -0,0 +1,11 @@
+import renderer from 'react-test-renderer';
+import Quote from '../../Components/Quote';
+
+describe('Quote component test', () => {
+  test('snapshot renders', () => {
+    const quote = renderer
+      .create(<Quote />)
+      .toJSON();
+    expect(quote).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] one of your tests is failing. Kindly fix that

![image](https://user-images.githubusercontent.com/98453979/207870579-81b21b63-0120-4439-a024-2aba299cc33f.png)